hedera-node/src/main/java/com/hedera/services/utils/UnzipUtility.java

```diff
@@ -68,14 +68,17 @@ public static void unzip(byte[] bytes, String dstDirectory) throws IOException {
 		ZipEntry entry = zipIn.getNextEntry();
 		while (entry != null) {
 			String filePath = dstDirectory + File.separator + entry.getName();
+			File fileOrDir = new File(filePath);
```

Comment:
I prefer to use final variables where possible since it aids in reading code. With non-final vars, I don't know if they get modified later in the code or not. If they're final, then I know they don't get changed anywhere else.

Reply:
Yes, changed as suggested.